simulariumio/tests/converters/test_trajectory_converter.py

```diff
@@ -59,1011 +62,1180 @@ def mixed_agents_wrong_display_type2():
 
 
 def mixed_agents_invalid_agent_id():
+    # Agent IDs are larger than a 32 bit int can represent
     result = mixed_agents()
     result.agent_data.unique_ids[0][0] = MAX_AGENT_ID + 1
     return result
 
 
+# 3 default agents (radius 5-10) at given positions for 3 frames,
+# test string for plots
+default_agents_trajectory = three_default_agents()
+default_agents_converter = TrajectoryConverter(default_agents_trajectory)
+default_agents_data = JsonWriter.format_trajectory_data(default_agents_converter._data)
+
+# 2 default agents (radius 5-10) and 3 fiber agents
+# at given positions for 3 frames, no plots
+mixed_agents_trajectory = mixed_agents()
+mixed_agents_converter = TrajectoryConverter(mixed_agents_trajectory)
+mixed_agents_data = JsonWriter.format_trajectory_data(mixed_agents_converter._data)
+
+# 3 fiber agents with points drawn
+# at given positions for 3 frames, no plots
+fiber_agents_trajectory = fiber_agents()
+fiber_agents_converter = TrajectoryConverter(fiber_agents_trajectory)
+fiber_agents_data = JsonWriter.format_trajectory_data(fiber_agents_converter._data)
+
+
+# test trajectory info
 @pytest.mark.parametrize(
-    "trajectory, expected_data",
+    "trajectory_version, expected_version",
+    [
+        (
+            default_agents_data["trajectoryInfo"]["version"],
+            CURRENT_VERSION.TRAJECTORY_INFO,
+        ),
+        (
+            mixed_agents_data["trajectoryInfo"]["version"],
+            CURRENT_VERSION.TRAJECTORY_INFO,
+        ),
+        (
+            fiber_agents_data["trajectoryInfo"]["version"],
+            CURRENT_VERSION.TRAJECTORY_INFO,
+        ),
+    ],
+)
+def test_versions_trajectory(trajectory_version, expected_version):
+    assert trajectory_version == expected_version
+
+
+# test time units
+@pytest.mark.parametrize(
+    "timeUnits, expected_timeUnits",
+    [
+        (
+            default_agents_data["trajectoryInfo"]["timeUnits"],
```

Comment:
I think the default agents should not include any data that has a default. That way you can test both that the defaults work and that changing the data works. It would also be nice to pull in the default magnitude and default name here